### PR TITLE
fix: [#9666] Onclick Insert property reference in a memory drop down not appearing in Bot Framework Composer 2.1.3.

### PR DESCRIPTION
**Pull request by JhontSouth**
#minor

## Description
This PR adds the CSRF token as default in the request of project variables using _axios HttpClient_ instead of _fetch_.

## Task Item

Closes # 9666

## Screenshots
The following image shows the references dropdown working after the fix.
image

### CODE REVIEW

File: Composer/packages/client/src/recoilModel/dispatchers/utils/project.ts [hunk 1]
```diff
@@ -912,7 +912,6 @@ export const checkIfBotExistsInBotProjectFile = async (
 };
 
 export const getMemoryVariables = async (projectId: string, options?: { signal: AbortSignal }) => {
-  const res = await fetch(`${BASEURL}/projects/${projectId}/variables`, { signal: options?.signal });
-  const json = (await res.json()) as { variables: string[] };
-  return json.variables ?? [];
+  const res = await httpClient.get(`/projects/${projectId}/variables`, { signal: options?.signal });
```
Comment (ceciliaavila) [inline Composer/packages/client/src/recoilModel/dispatchers/utils/project.ts:915]: The BASEURL variable is no longer needed?

Comment (JhontSouth) [reply Composer/packages/client/src/recoilModel/dispatchers/utils/project.ts:915]: No, the content of BASEURL is '/api', but the httpClient add it by default 
image

**Review by ceciliaavila — APPROVED** [review]

**Comment by tracyboehrer** [comment]
This isn't opened on the microsoft/botframework-composer repo.  Was this intentional?

**Comment by JhontSouth** [comment]
> This isn't opened on the microsoft/botframework-composer repo. Was this intentional?

Sorry @tracyboehrer, this is our internal PR, we'll promote it to MS once it's reviewed by the team.

**Review by sw-joelmut — APPROVED** [review]

**Comment by JhontSouth** [comment]
Promoted /9678

**Review by ceciliaavila — APPROVED** [review]

**Pull request closed without merging**